.vscode/launch.json

```diff
@@ -0,0 +1,71 @@
+{
+    "configurations": [
+        {
+            "name": "Debug cloe",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${input:get_executable_path}",
+            "args": [
+                "run",
+                "--no-write-output",
+                //example below shows how to use a specific test-case
+                //"${workspaceFolder}/tests/test_lua04_schedule_test.lua",
+                "${workspaceFolder}/${input:get_test_case}",
+            ],
+            "stopAtEntry": false,
+            "cwd": "${fileDirname}",
+            "environment": [],
+            "externalConsole": false,
+            "MIMode": "gdb",
+            "setupCommands": [
+                {
+                    "description": "Enable pretty-printing for gdb",
+                    "text": "-enable-pretty-printing",
+                    "ignoreFailures": true
+                },
+                {
+                    "description": "Set Disassembly Flavor to Intel",
+                    "text": "-gdb-set disassembly-flavor intel",
+                    "ignoreFailures": true
+                }
+            ],
+            "preLaunchTask": "build cloe",
+            "miDebuggerPath": "/usr/bin/gdb",
+            "envFile": "${input:get_shell_path}"
+        }
+    ],
+    "inputs": [
+        {
+            "type": "command",
+            "id": "get_test_case",
+            "command": "shellCommand.execute",
+            "args":{
+                "command":"ls tests/*.lua",
+                "cwd": "${workspaceFolder}"
+            }
+        },
+        {
+            "type": "command",
+            "id": "get_shell_path",
+            "command": "shellCommand.execute",
+            //bash magic to set CLOE_SHELL variable and get the value of it
+            "args": {
+                "command": "bash -c \"source <(cloe-launch activate tests/conanfile_all.py) && printenv CLOE_SHELL\"",
```

Comment:
Use `python3 -m cloe_launch` with environment variable `PYTHONPATH` set to repo `cli`.